<polymer-element name="pathiways-home" noscript>
    <template>
        <style>
            :host {
                display: block;
                position: relative;
                box-sizing: border-box;
                z-index: 50000;

                margin: 0px auto 0px auto;

                width: 70%;
                max-width: 780px;
                color: #FFF;
                font-size: 22px;
                line-height: 25px;
                font-weight: 300;
            }

            .paragraph {
                text-align: justify;
                padding: 10px 0;
            }
            .paragraph2 {
                font-size: 16px;
                padding: 0 0 10px 0;
            }

            .title {
                font-size: 40px;
                padding-top: 30px;
                padding-bottom: 10px;
                color: white;
            }

            .title2 {
                color: white;
                padding-top: 20px;
                padding-bottom: 10px;
            }

            .b {
                font-weight: 400;
            }

            a, a:visited {
                text-decoration: none;
                color: #ffad76;
            }
        </style>
        <!--SUITE_INFO = '<div style=" width: 800px;">'-->
        <!--+ '<h4>Overview</h4><br>'-->
        <!--+ '<h3>PATHiWAYS</h3>'-->
        <!--+ '<p align="justify">PATHiWAYS is a web tool for the interpretation of the consequences of the combined changes in expression levels of genes in the context of signaling pathways. Specifically, this tool allows the user to identify the stimulus-response subpathways that are significantly activated or deactivated in the typical case/control experiment. PATHiWAYS identifies all the stimulus-response subpathways of KEGG signaling pathways, calculates the probability of activation of each one, based on the individual gene expression values and identifies those with a significant differential activity between the two conditions compared.</p>'-->
        <!--+ '<br>'-->
        <!--+ '<h3>PATHiPRED</h3>'-->
        <!--+ '<p align="justify">The aim of PATHiPRED web tool is to provide a classifier that takes advantage of the activity values of signal transduction along the elementary components of signaling pathways. PATHiPRED differentiates between two classes or a continuous variable using activation probabilities of stimulus-response sub-pathways calculated by PATHiWAYS methodology. PATHiPRED performs a SVM modelling with cross-validation and the results include the prediction model, the statistical parameters that assess the goodness of the model, the confusion matrix of the prediction, the activation probabilities matrix, the mechanism-based biomarkers and pathways graphs with the selected sub-pathways highlighted. Moreover, the prediction model obtained can be applied to a new dataset within PATHiPRED web tool.</p>'-->
        <!--+ '<br>'-->
        <!--+ '<p align="justify"><h4>Note</h4><br>This web application makes an intensive use of new web technologies and standards like HTML5, so browsers that are fully supported for this site are: Chrome 14+, Firefox 7+, Safari 5+ and Opera 11+. Older browser like Chrome13-, Firefox 5- or Internet Explorer 9 may rise some errors. Internet Explorer 6 and 7 are no supported at all.</p>'-->
        <!--+ '</div>';-->
        <div class="title">Overview</div>
        <!--<p class="title2">PATHiWAYS</p>-->

        <!--<div class="paragraph">PATHiWAYS is a web tool for the interpretation of the consequences of the combined-->
        <!--changes-->
        <!--in expression levels of genes in the context of signaling pathways. Specifically, this tool allows the-->
        <!--user-->
        <!--to identify the stimulus-response subpathways that are significantly activated or deactivated in the-->
        <!--typical-->
        <!--case/control experiment. PATHiWAYS identifies all the stimulus-response subpathways of KEGG signaling-->
        <!--pathways, calculates the probability of activation of each one, based on the individual gene expression-->
        <!--values and identifies those with a significant differential activity between the two conditions-->
        <!--compared.</div>-->

        <!--<p class="title2">PATHiPRED</p>-->

        <!--<p class="paragraph">The aim of PATHiPRED web tool is to provide a classifier that takes advantage of the activity-->
        <!--values of signal transduction along the elementary components of signaling pathways. PATHiPRED differentiates-->
        <!--between two classes or a continuous variable using activation probabilities of stimulus-response sub-pathways-->
        <!--calculated by PATHiWAYS methodology. PATHiPRED performs a SVM modelling with cross-validation and the results-->
        <!--include the prediction model, the statistical parameters that assess the goodness of the model, the confusion-->
        <!--matrix of the prediction, the activation probabilities matrix, the mechanism-based biomarkers and pathways graphs-->
        <!--with the selected sub-pathways highlighted. Moreover, the prediction model obtained can be applied to a new-->
        <!--dataset within PATHiPRED web tool.</p>-->


        <!--<p class="title2">PATHiVAR</p>-->

        <div class="paragraph">
            PATHiVAR estimates the functional impact that mutations have over the human signalling network.
        </div>

        <div class="paragraph">
            PATHiVAR analyses VCF files, extract the deleterious mutations, locate them over the signalling
            pathways in the selected tissue (with the appropriate expression pattern) and provides a
            comprehensive, graphic and interactive view of the predicted signal transduction probabilities
            across the different signalling pathways.
        </div>

        <div class="title2">Note</div>
        <div class="paragraph2">
            This web application makes an intensive use of new web technologies and standards like HTML5,
            so only the browser Chrome 39 is fully supported. we are working hard to make it compatible with other browsers.
        </div>
        <!--<p style="text-align:justify;font-size: 16px">This web application makes an intensive use of new web technologies and standards-->
        <!--like HTML5, so browsers that are fully supported for this site are: Chrome 14, Firefox 7, Safari 5 and Opera 11. Older-->
        <!--browser like Chrome13-, Firefox 5- or Internet Explorer 9 may rise some errors. Internet Explorer 6 and 7 are no supported-->
        <!--at all.</p>-->
    </template>
</polymer-element>
